src/LibraryViewExtension/packages.config

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="cef.redist.x64" version="3.2623.1401" targetFramework="net45" />
-  <package id="cef.redist.x86" version="3.2623.1401" targetFramework="net45" />
-  <package id="CefSharp.Common" version="49.0.1" targetFramework="net45" />
-  <package id="CefSharp.Wpf" version="49.0.1" targetFramework="net45" />
+  <package id="cef.redist.x64" version="3.2883.1552" targetFramework="net45" />
+  <package id="cef.redist.x86" version="3.2883.1552" targetFramework="net45" />
+  <package id="CefSharp.Common" version="55.0.0" targetFramework="net45" />
+  <package id="CefSharp.Wpf" version="55.0.0" targetFramework="net45" />
+  <package id="Greg" version="1.0.6176.18754" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
-  <package id="RestSharp" version="105.2.3" targetFramework="net45" />
+  <package id="RestSharp" version="105.2.3" targetFramework="net45" requireReinstallation="true" />
```

Comment:
What does this `requireReinstallation="true"` do?

Reply:
this happened as a result of changing to .net 4.5.2 apparently it will help the solution build on machines where this dep was already installed.